<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CautionsWarningsDialog</class>
 <widget class="QDialog" name="CautionsWarningsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Cautions and Warnings</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="3" colspan="2">
    <widget class="QPushButton" name="searchButton_">
     <property name="text">
      <string>&amp;Search</string>
     </property>
     <property name="shortcut">
      <string>Alt+S</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="helpButton_">
     <property name="text">
      <string>&amp;Help</string>
     </property>
     <property name="shortcut">
      <string>Alt+H</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="cancelButton_">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
     <property name="shortcut">
      <string>Alt+C</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="xincludeButton_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Insert Link</string>
     </property>
     <property name="shortcut">
      <string>Alt+I</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QPushButton" name="linkButton_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Link and Adapt</string>
     </property>
     <property name="shortcut">
      <string>Alt+L</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <spacer name="Horizontal Spacing2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>290</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="searchTextLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Search for standard texts:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="searchLineEdit_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBox_">
     <property name="text">
      <string>Insert standard text only at cursor location</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTreeWidget" name="listView_">
     <property name="styleSheet">
      <string notr="true">QTreeWidget::branch{image:none;}</string>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Text</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="5" rowspan="3" colspan="3">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEdit_"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QGroupBox" name="filterGroupBox_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="4">
       <widget class="QRadioButton" name="textRadioButton_">
        <property name="text">
         <string>texts</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filterGroup_</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="noteRadioButton_">
        <property name="text">
         <string>notes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filterGroup_</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="warnRadioButton_">
        <property name="text">
         <string>warnings</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filterGroup_</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="cauRadioButton_">
        <property name="text">
         <string>cautions</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filterGroup_</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton17">
        <property name="text">
         <string>All</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filterGroup_</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>searchLineEdit_</tabstop>
  <tabstop>helpButton_</tabstop>
  <tabstop>linkButton_</tabstop>
  <tabstop>xincludeButton_</tabstop>
  <tabstop>cancelButton_</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>helpButton_</sender>
   <signal>clicked()</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton_</sender>
   <signal>clicked()</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linkButton_</sender>
   <signal>clicked()</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>insertLink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xincludeButton_</sender>
   <signal>clicked()</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>insertXInclude()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchLineEdit_</sender>
   <signal>returnPressed()</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton_</sender>
   <signal>clicked()</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterGroup_</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>CautionsWarningsDialog</receiver>
   <slot>filterChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="filterGroup_"/>
 </buttongroups>
</ui>
